mojo/stdlib/benchmarks/README.md

```diff
@@ -17,31 +17,44 @@ internally.
 
 ## How to run the benchmarks
 
-If you want to just compile and run all of the benchmarks as-is,
-there is a script provided [here](../../stdlib/scripts/run-benchmarks.sh).
+If you want to just compile and run all the benchmarks as-is,
+we need to execute the following command:
+
+```bash
+./bazelw test mojo/stdlib/benchmarks/... --local_test_jobs=1 --test_output=all
+```
+
 This script builds the open source `stdlib.mojopkg` and then executes
-all of the benchmarks sequentially. The script also allows specifying a
-subdirectory or a file to run.
+all the benchmarks sequentially.
+
+If you wish to test changes you are making on the current branch, remove the `-t` flag on top
+of the `mojo/stdlib/benchmarks/BUILD.bazel` BAZEL file.
+
+To run a specific benchmark, you need to change the following line BAZEL file:
+
+```bash
+for src in glob(["**/*.mojo"])
+```
+
+To something like this:
+
+```bash
+for src in glob(["**/bench_list.mojo"])
+```
 
-Running e.g. `pixi run mojo run stdlib/benchmarks/collections/bench_dict.mojo`
-makes the linker use the existing branch that the compiler is on. If you wish to
-test changes you are making on the current branch, remove the `-t` flag on top
-of the benchmark file (`# RUN: %mojo-no-debug %s -t`) then run:
-`pixi run stdlib/scripts/run-benchmarks.sh
-stdlib/benchmarks/collections/bench_dict.mojo`.
-Remember to replace the `-t` flag again before pushing any code.
+Remember to revert the `-t` flag and the `glob` changes again before pushing any code.
 
 ## How to write effective benchmarks
 
-All of the benchmarks use the `benchmark` module.  `Bench` objects are built
-on top of the `benchmark` module.  You can also use `BenchConfig` to configure
+All the benchmarks use the `benchmark` module. `Bench` objects are built
+on top of the `benchmark` module. You can also use `BenchConfig` to configure
 `Bench`. For the most part, you can copy-paste from existing
 benchmarks to get started.
 
 ## Benchmarks in CI
 
 Currently, there is no short-term plans for adding these benchmarks with regression
-detection and such in the public Mojo CI.  We're working hard to improve the processes
+detection and such in the public Mojo CI. We're working hard to improve the processes
```

Comment:
```suggestion
detection and such in the public CI. We're working hard to improve the processes
```
as a drive-by, perhaps drop "Mojo" now that the Mojo and MAX repos are merged and named `modular`?